elasticsearch/dsl/connections.py

```diff
@@ -116,16 +116,16 @@ def get_connection(self, alias: Union[str, _T] = "default") -> _T:
             raise KeyError(f"There is no connection with alias {alias!r}.")
 
     def _with_user_agent(self, conn: _T) -> _T:
-        from . import __versionstr__  # this is here to avoid circular imports
+        from elasticsearch import (
+            __versionstr__,  # this is here to avoid circular imports
```

Comment:
Technically, this is not a circular import, as `elasticsearch` does not import `elasticsearch.dsl` today.

Reply:
Correct, this can be moved to the top of the file.